scss/_patterns_search-box.scss

```diff
@@ -38,7 +38,7 @@
     @extend %transparent-button;
 
     &:not(:last-of-type):not(:only-of-type) {
-      margin-right: $bar-thickness;
+      margin-right: $bar-thickness !important;
```

Comment:
Does this need important?

Reply:
I don't mean for this to be reviewed yet, I'm using Percy to show me what effect changes are having across the examples.

However, it seems like !important will need to be used somewhere - I had them originally on the base button styles like the codepen, but that overrode the search box button margin. I removed it from the base styles, but found that the `:last-child` rule still took precedence, so added !important here to work around that.

Reply:
Ugh - turns out this is not where I meant to add the rule anyway...!